beam-llm-hosting/shared/answer_from_model.py

```
@@ -24,13 +26,23 @@ def answer_from_model(model, tokenizer, incoming_messages: List[Dict[str, str]],
-    output = outputs[len(outputs) - 1]
-    outputs = output.split(incoming_messages[len(incoming_messages) - 1]["content"])
+    if llm_result == "":
+        raise Exception("No answer from model")
+    last_message = incoming_messages[len(incoming_messages) - 1]["content"]
+    outputs = llm_result.split(last_message)
+    strpos = 0
+    while len(outputs) == 1:
+        # something was changed in the last message, likely dot or comma placement corrections
+        strpos += 1
+        if strpos >= len(last_message):
+            raise Exception("Can't find last message in output.")
```

Ruff (0.13.1)

Create your own exception

(TRY002)